patches/lua-tree.patch

```diff
@@ -183,7 +189,7 @@ index 625bb0c63..956a09ec9 100644
 -    return re_find(value, uuid_regex, "ioj") and true or nil
 +    return re_match(value, uuid_regex) and true or nil
    end,
- 
+
```

Comment:
Why are there so many unrelated changes here?

Reply:
These are just whitespaces strip, not sure why the rebuild is adding them. I'll clean this up

Reply:
There are also code changes. Look at the line 184/190.

Reply:
Those diffs are part of patch itself and are misleading due to the way these are visually rendered in github: for example line [190](https://github.com/Kong/goks/blob/0588232eac5663a6e0770e624ef16c58374de0b4/patches/lua-tree.patch#L190)